Slider/Assets/Scripts/Map/Interactable/Collectible.cs

```diff
@@ -80,6 +80,7 @@ private void OnTriggerEnter2D(Collider2D collision)
     public void ActivateSTile(int stileId) 
     {
         SGrid.Current.CollectSTile(stileId);
+        AchievementManager.IncrementAchievementStat("slidersCollected");
```

Comment:
can we make this call to set them equal instead of incrementing? esp since we have a method for getting the number of sliders in an area already

also side note we prob will have to do inventory collectible loops later eventually, for the breadge 3 achiev (loop thru collectibles and check collectible names)